### DISCUSSION TITLE
Citrate synthase assay feasibility for undergrad student

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1652] [General]
I have an undergraduate intern this summer (biochemistry major) and I'm considering having her use a citrate synthase assay for some crab samples. How difficult was it to optimize the assay for oysters, and would you recommend it for an undergraduate student with minimal research experience? @ocattau @mattgeorgephd

**Comment by ocattau** [comment]
I think it is something an undergrad can do it, especially if they have the whole summer to do it. Good lab skills are necessary but it is not so advanced someone can't gain the skill. Do you need my protocol?

**Reply by yaaminiv** [reply]
protocol would be great! were there specific things you found difficult/was there a lot of troubleshooting?

**Reply by ocattau** [reply]
here you go! https://docs.google.com/document/d/1Z5N3Hiwnssrt99567thmBYMuz3mTyUL3P3Gfe8bJXNw/edit

Protein Standardization is something we decided to do in order to speed up the procedure and normalize as best as possible to prevent large standard deviations. So we stacked the CS protocol on top of the BSA protein protocol. The one thing I found problematic was the amount of bubbles that form when you homogenize the tissue in the buffer as the bubbles makes the spec readings less accurate.
